templates/components/page-title-section.tpl

```diff
@@ -5,5 +5,5 @@
 {$componentName = 'page-title-section'}
 
 {block name='page_title_section'}
-  <h1 class="{$componentName} mb-4">{$title}</h1>
+  <h1 class="{$componentName} h2 mb-4">{$title}</h1>
```

Comment:
If even the page title doesn't deserve class `h1` then maybe we should adjust the styling of `.h1`

Reply:
the idea is to keep the hierarchy and styling apart, it looks good